scipy/special/boost_special_functions.h

```diff
@@ -760,18 +760,22 @@ ncf_pdf_double(double x, double v1, double v2, double l)
 
 template<typename Real>
 Real
-ncf_cdf_wrap(const Real x, const Real v1, const Real v2, const Real l)
+ncf_cdf_wrap(const Real v1, const Real v2, const Real l, const Real x)
```

Comment:
Could you explain what is going on here? `ncf_cdf_float` and `ncf_cdf_double` are calling this function with arguments like `ncf_cdf_wrap(x, v1, v2, l)`, but it looks like these are ordered differently. Also, the arguments accepted by `ncf_cdf_float` and `ncf_cdf_double` appear to be in a different order.

Reply:
Looks like a mistake on my part. Because they were worked on separately, `ncfdtr` and the boost wrapping ufunc `_ncf_cdf` take a different order for the arguments, and I messed up trying to standardize on the `ncfdtr‘ order, but I guess in such a way that my mistakes cancelled out and the tests still pass. Not at a computer now, but I’ll fix this.

Reply:
@steppi : could you add a short sentence about the discrepancy in the argument order between `stats.nct.cdf` and `special.nctdtr`? Such things are for sure a constant source of confusion for users if maintainers already are confused themselves.